main.go

```diff
@@ -789,6 +868,15 @@ func (s *Server) injectRoute(path *bgptable.Path) error {
 		log.Printf("removed route %s from kernel", nlri)
 		return netlink.RouteDel(route)
 	}
+	if !ipip {
+		// Recursive nexthop lookup for GCE environment
```

Comment:
Yes, it is also valid for other cloud environments.

The comment was misleading. I updated the comment. PTAL